package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
-    "google-sheet-cli": "^1.6.0"
+    "google-sheet-cli": "^1.6.0",
+    "react-error-boundary": "^4.0.11"
```

Comment:
Any reason the `react-error-boundary` is added through root level?

This should be in `ui` to avoid confusion, and also this entire repo isn't a true monorepo, so we're not using this `package.json` at the root level.

Reply:
oh ,thanks for points , its by mistake .